python/GafferUITest/TestCase.py

```diff
@@ -83,6 +83,29 @@ def assertExampleFilesExist( self ) :
 			expanded = os.path.expandvars( e['filePath'] )
 			self.assertTrue( os.path.exists( expanded ), "%s does not exist" % expanded )
 
+	def assertExampleFileDontReferenceUnstablePaths( self ) :
```

Comment:
assertExampleFileDontReferenceUnstablePaths -> assertExampleFile**s**DontReferenceUnstablePaths

Reply:
Doh! Sorry. Fixed in https://github.com/GafferHQ/gaffer/pull/3309/commits/549381d8a6c0d876247ecc5e2dbe18014584ba18